src/abaqus/BoundaryCondition/BoundaryConditionModel.py

```diff
@@ -794,7 +795,7 @@ def DisplacementBC(
         self,
         name: str,
         createStepName: str,
-        region: Region,
+        region: typing.Union[Region,Set],
```

Comment:
Why a Set object can be accepted in these methods

Reply:
That came right from the Tutorial. Look, in the snippet:
```python
bottom_instance = model.rootAssembly.instances['instance'].sets['set-bottom']
bc = model.DisplacementBC(name='BC-1', createStepName='Initial',
                          region=bottom_instance, u3=SET)
```
The `bottom_instance ` object is a `Set`

Reply:
I think there are some errors in the class inheritances of the Set and Surface, according to the description of the [Region](https://pyabaqus.haiiliin.com/en/latest/reference/mdb/model/part_assembly/region.html?highlight=Region#id1) class, the Region class should be base class of the Set and Surface. Then we just need Region here.

Reply:
Create a pull request in [haiiliin/abqpy#259](https://github.com/haiiliin/abqpy/pull/259) related to this issue.